languages/swift/exercises/concept/arrays/.docs/instructions.md

```diff
@@ -35,7 +35,7 @@ insert(newCard, atTopOf: [5, 9, 7, 1])
 
 ## 4. Remove a card from the stack
```

Comment:
For the remove from bottom / from top exercises: What should happen in case the array was already empty (returning an empty array?)? I would specify this case as well to have it more precise.

All tasks using an index would be a similar question: What happens, if the index does not exist? Here I would be also fine to let it unspecified (and assuming students will not handle this case) and having the built-in runtime exception be thrown in this case.

Reply:
Good point. I will amend the exercise to make sure thee relevant check is made and add the relevant tests.